app/components/TransactionHistory/TransactionHistoryPanel/Transactions.jsx

```diff
@@ -21,10 +22,7 @@ export default class Transactions extends React.Component<Props> {
 
   render() {
     const { className, transactions, pendingTransactions } = this.props
-    if (
-      !transactions.length === 0 &&
-      (!pendingTransactions || !pendingTransactions.length)
-    ) {
+    if (isEmpty(transactions) && isEmpty(pendingTransactions)) {
```

Comment:
I think `!transactions.length === 0` should be `!isEmpty(transactions)`

Reply:
`if (isEmpty(transactions) && isEmpty(pendingTransactions))` should work